windows/src/engine/keyman32/RightAltEmulationCheck.cpp

```diff
@@ -65,7 +65,7 @@ typedef PKBDTABLES_WOW64 (WINAPI *PKBDLAYERDESCRIPTORWOW64FUNC)(VOID);
 #endif
 
 HMODULE LoadKbdLibrary(const char* keyboardLayoutName);
-BOOL ReadAltGrFlagFromKbdDll(const char *keyboardLayoutName);
+BOOL ReadAltGrFlagFromKbdDll(const char *keyboardLayoutName, BOOL& result);
```

Comment:
In diagnosing this issue, I split function success out from result, which showed that the function wasn't failing in the unit test. Left this change in as it is more correct, even though we ignore the success value in the one place it is checked (not much else we can do there).